hardpy/pytest_hardpy/pytest_call.py

```diff
@@ -26,6 +27,46 @@ class CurrentTestInfo(object):
     case_id: str
 
 
+class DialogBoxWidgetType(Enum):
+    """Dialog box widget type."""
+
+    RADIOBUTTON = "radiobutton"
+    CHECKBOX = "checkbox"
+    TEXT_INPUT = "textinput"
+    NUMERIC_INPUT = "numericinput"
+
+
+@dataclass
+class DialogBoxWidget:
+    """Dialog box widget."""
+
+    widget_type: DialogBoxWidgetType
+    widget_info: str | dict | float
+
+
+@dataclass
+class DialogBoxData:
+    """Dialog box data.
+
+    Args:
+        title_bar (str): title bar
+        dialog_text (str): dialog text
+        widget_info (DialogBoxWidget | None): widget info
+    """
+
+    title_bar: str
+    dialog_text: str
+    widget_info: DialogBoxWidget | None
```

Comment:
Need to move in `dialog_box.py` in utils dir.

Reply:
Done, but I'm not sure the import is done correctly

Reply:
Ready